analysed-packages/glib-networking/version-2.72.2/glib-networking-2.72.2-OSS-disclosure.txt

```diff
@@ -0,0 +1,2081 @@
+to the public, we recommend making it free software that everyone can
+redistribute and change. You can do so by permitting redistribution under these
+terms (or, alternatively, under the terms of the ordinary General Public
+License).
+
+To apply these terms, attach the following notices to the library. It is safest
+to attach them to the start of each source file to most effectively convey the
+exclusion of warranty; and each file should have at least the "copyright" line
+and a pointer to where the full notice is found.
+
+<one line to give the library's name and an idea of what it does.>
+
+Copyright (C) <year> <name of author>
+
+This library is free software; you can redistribute it and/or modify it under
+the terms of the GNU Lesser General Public License as published by the Free
+Software Foundation; either version 2.1 of the License, or (at your option) any
+later version.
+
+This library is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this library; if not, write to the Free Software Foundation, Inc., 51
+Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+Also add information on how to contact you by electronic and paper mail.
+
+You should also get your employer (if you work as a programmer) or your school,
+if any, to sign a "copyright disclaimer" for the library, if necessary. Here is
+a sample; alter the names:
+
+Yoyodyne, Inc., hereby disclaims all copyright interest in
+
+the library `Frob' (a library for tweaking knobs) written
+
+by James Random Hacker.
+
+< signature of Ty Coon > , 1 April 1990
+
+Ty Coon, President of Vice
+
+That's all there is to it!
+
+In addition, as a special exception, the copyright holders give permission to
+link the code of portions of this program with the OpenSSL library under certain
+conditions as described in each individual source file, and distribute linked
+combinations including the two.
+
+You must obey the GNU General Public License in all respects for all of the code
+used other than OpenSSL. If you modify file(s) with this exception, you may
+extend this exception to your version of the file(s), but you are not obligated
+to do so. If you do not wish to do so, delete this exception statement from your
+version. If you delete this exception statement from all source files in the
+program, then also delete it here.
+
+--------------------------------------------------------------------------------
+
+License-of-GNU-Licenses
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+--------------------------------------------------------------------------------
+
+RSA-Cryptoki
```

Comment:
This is not the text which is available in the file pkcs11/pkcs11t.h. The text in the file is:

License to copy and use this software is granted provided that it is
identified as "RSA Security Inc. PKCS  11 Cryptographic Token Interface
(Cryptoki)" in all material mentioning or referencing this software.

License is also granted to make and use derivative works provided that
such works are identified as "derived from the RSA Security Inc. PKCS  11
Cryptographic Token Interface (Cryptoki)" in all material mentioning or
referencing the derived work.

RSA Security Inc. makes no representations concerning either the
merchantability of this software or the suitability of this software for
any particular purpose. It is provided "as is" without express or implied
warranty of any kind.

Please correct this.